integration/spark/src/test/scala/org/apache/spark/carbondata/TestStreamingTableQueryFilter.scala

```diff
@@ -96,25 +98,97 @@ class TestStreamingTableQueryFilter extends QueryTest with BeforeAndAfterAll {
     // filter
     checkAnswer(
       sql("select * from streaming_table_filter.stream_filter where id = 1"),
-      Seq(Row(1, "name_1", "city_1", 10000.0, BigDecimal.valueOf(0.01), 80.01, Date.valueOf("1990-01-01"), Timestamp.valueOf("2010-01-01 10:01:01.0"), Timestamp.valueOf("2010-01-01 10:01:01.0"), ("1" + longStrValue), Row(wrap(Array("school_1", "school_11")), 1))))
+      Seq(Row(1,
```

Comment:
can enable scala style off/on to avoid changes in this file

Reply:
done, changed for all places